scripts/docker-setup.sh

```diff
@@ -80,6 +80,11 @@ function setup_mountpoints() {
 function save_start_command() {
     url="https://raw.githubusercontent.com/automatic-ripping-machine/automatic-ripping-machine/main/scripts/docker/start_arm_container.sh"
     cd ~arm
+    if [ -e start_arm_container.sh ]
+    then
+        echo -e "'start_arm_container.sh' already exists.  Renaming old file to 'start_arm_container_OLD.sh'"
+        sudo mv /start_arm_container.sh ./start_arm_container_OLD.sh
```

Comment:
```suggestion
        sudo mv /start_arm_container.sh ./start_arm_container.sh.bak
```

Reply:
yeah, this is a better solution.  I'm too used to personal projects where additional jank like this is normal for me.